heron/spi/src/java/com/twitter/heron/spi/utils/SchedulerUtils.java

```diff
@@ -436,6 +436,10 @@ public static void persistUpdatedPackingPlan(String topologyName,
                                                SchedulerStateManagerAdaptor stateManager) {
     LOG.log(Level.INFO, "Updating scheduled-resource in packing plan: {0}", topologyName);
     PackingPlanProtoSerializer serializer = new PackingPlanProtoSerializer();
+    if (stateManager.getPackingPlan(topologyName) != null) {
+      stateManager.deletePackingPlan(topologyName);
```

Comment:
I've added update commands to the state manager adaptor. Also changes `AuroraScheduler` to not update state manager as a side effect of calling `createAuroraProperties`.